website/docs/docs/ai/index.md

```diff
@@ -33,6 +33,12 @@ For *nix users (including Windows users running WSL/msys2/mingw), use the follow
 curl https://download.pactflow.io/ai/get.sh | sh
 ```
 
+For Windows PowerShell users, use the following command to download and install:
+
+```pwsh
+Invoke-WebRequest -Uri "https://download.pactflow.io/ai/get.ps1" | Invoke-Expression
```

Comment:
```suggestion
Invoke-WebRequest -Uri https://download.pactflow.io/ai/get.ps1 | Invoke-Expression
```

Pretty sure the quoting is not needed (at least, it works for me fine in macOS)

Reply:
Good point. Thanks.